jdi-light-mobile/src/main/java/com/epam/jdi/mobile/driver/get/DriverData.java

```diff
@@ -104,32 +171,30 @@ public static OsTypes getOs() {
         if (groups.size() == 2)
             driver.manage().window().setSize(new Dimension(parseInt(groups.get(0)), parseInt(groups.get(1))));
         else {
-            if (getOs().equals(MAC))
+            if (getOs().equals(MAC) && !DRIVER_NAME.equals("appium")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4973196)